zipkin/src/main/java/zipkin/storage/InMemorySpanStore.java

```diff
@@ -17,14 +17,17 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.Comparator;
+import java.util.HashSet;
```

Comment:
always linkedhashset :) helps to avoid breaking tests or other things that look at order. less distractions